tests/by-util/test_more.rs

```diff
@@ -32,3 +38,15 @@ fn test_more_dir_arg() {
             .usage_error("'.' is a directory.");
     }
 }
+
+#[test]
+#[cfg(target_family = "unix")]
+fn test_more_invalid_file_perms() {
+    if std::io::stdout().is_terminal() {
+        let (at, mut ucmd) = at_and_ucmd!();
+        let permissions = Permissions::from_mode(0o244);
+        at.make_file("invalid-perms.txt").metadata().unwrap();
+        set_permissions(at.plus("invalid-perms.txt"), permissions).unwrap();
+        ucmd.arg("invalid-perms.txt").fails();
```

Comment:
The `fails()` alone is not enough because currently the test passes even without applying your changes. I would enable the output check you had in a previous commit.

Reply:
ah yes looking at the cp tests i found stderr_contains. thats what i was missing :)